content/rule_book/quiz_process.md

```diff
@@ -1,106 +1,173 @@
 - If a quizzer's light comes on after the quizmaster has discernibly begun reading the question, this will be considered a valid jump, and the quizzer will be given an opportunity to answer.
-- The first quizzer to rise and trigger their light, provided there has been no foul, will be awarded the jump.
+- The 1st quizzer to rise and trigger their light, provided there has been no foul, will be awarded the jump.
 - If in the judgment of the officials, a deliberate attempt is made to forfeit a question, an error will be charged, and 10 points deducted from the team score. The next question will be the same question number.
 
-### Tie Jumping
+## Timeliness
 
-- When members of the same team tie, the captain will select one quizzer to answer without any type of verbal or non-verbal communication.
-- When quizzers of two or three teams tie, the question will be replaced with a question of the same type on which all eligible members of the tied teams may jump. The question number remains the same.
-- When a quizzer errs on the tie-jump question, it will be treated as a regular question and the quiz will continue as normal.
+Except for the exception below:
 
-## Time Limits
+- Quizzes will start at the scheduled or announced time.
+- Any team that is late does not receive the starting 20 points awarded for being on time.
+- Any team more than 10 minutes late forfeits the quiz.
 
-- Quizzes will start at the scheduled or announced time. Any team that is late does not receive the starting 20 points awarded for being on time.
-    - If the late team is currently quizzing in another location which is running over the time limit, they are not required to forfeit the quiz or lose 20 points.
-    - Any team more than 10 minutes late (after the scheduled or announced time for the quiz) forfeits the quiz.
-- If an individual quizzer is late, the quiz will start on time. The coach has the option to put the late quizzer in during a timeout, but the team will forfeit the 20 points the team starts with. The coach also has the option to keep the quizzer out of the quiz and keep the 20 points but must notify the quizmaster at the beginning of the quiz, of their decision.
-- Quizzers have five seconds in which to jump after the question is completed.
+The exception to the above is if the late team was late due to quizzing in another location which is running late, they are not required to forfeit the quiz or lose 20 points.
 
-A quizzer has 30 seconds to complete their answer after being recognized.
+If any number of members of a team is late, the quiz will start on time. The coach has the option to put quizzers in during a timeout, but the team will then forfeit the 20 points the team starts with. The coach also has the option to keep the quizzer out of the quiz and keep the 20 points but must notify the quizmaster at the beginning of the quiz of their decision.
```

Comment:
"has the option to put _late_ quizzers in during a timeout." And, "has the option to keep the _late_ quizzer out of the quiz."

Also, this rule is unclear as to what happens if the quizzer shows up late, but the coach decides NOT to put them in, but also didn't notify the QM at the beginning of the quiz. Are the 20 points still forfeited? If not, why does the rule say the points are only forfeited at the point of subbing in? The points should instead be forfeited or not forfeited right when the QM is notified.

Reply:
Agreed. This is confusing. I think this is another current rule book bug, and we'll need to do a little assumptions of original intent to get at a clear writing for v1.

I suspect but cannot prove the intent of the "notify the QM" clause is to prevent strategy shifts mid-quiz in close quizzes. For example, let's say team's A and B are close in capability. Team B has an absent quizzer. Coach B says nothing about it, so the 20 starting points stand. The quiz gets to question 16 with A and B close in score, so team A presses speed slightly and subsequently increases their error probability. The missing quizzer shows up, and then coach B decides to substitute the quizzer in because losing the 20 points at that point in the quiz. Had coach B "notified the QM" (which is to say, made it known to all that there would be a sub-in of the late quizzer), that would change team A's strategy. (Also, in this example, please ignore the fact that team A changing their speed slightly was probably a bad idea strategically regardless of B's situation.)

So here's where I'm at: What we have in the v1 right now is FE to the current rule book. But, like Jeremy points out, there's a bug here. I think the way we fix this is as follows:

Prior to the start of a quiz, if not all quizzers are in the room, 1 of 2 options exist for the team's coach:

1. The coach submits a lineup with the quizzer missing, which means the late-arriving quizzer cannot be subbed-in
2. **OR** the coach submits a lineup with the quizzer added, tells *all* the officials *and* coaches the quizzer is late, and loses the 20 starting points

This is not FE, but I think it patches the bug, and I'd argue it's close to what happens now, plus it's fair.

Reply:
I agree. The coach should say at the beginning of the quiz if they intend to sub in missing quizzers or not.

Reply:
I don't think this would be a controversial area to make a small fix, just to get rid of a bug. So this has my support, definitely one to mark though because it won't be FE.

Reply:
I agree. I think that makes the most sense. One decision to make at the beginning of the quiz.

Reply:
dc4347e